.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -99,6 +99,12 @@ We should update the README to:
 - [ ] Update the "Build and Run Instructions" with any new dependencies.
       Check for changes in the `Dockerfile` since the last tag: `git diff <previous-release-tag> docker/Dockerfile`.
 
+## Checkpoints
+
+With every release and for performance reasons, we want to update the zebra checkpoints. More information on how to do this can be found in [the zebra-checkpoints README](https://github.com/ZcashFoundation/zebra/blob/main/zebra-consensus/src/checkpoint/README.md).
+
+To do this you will need a synchronized `zcashd` node. You can request help from other zebra team members to submit this PR if you can't make it yourself at the moment of the release.
```

Comment:
@teor2345 i don't update the checkpoints since a while but can this be done with zebra as well ?

Reply:
`zebra-checkpoints` uses the `getblockcount` RPC, which we haven't implemented yet.

We could switch over to `getblockchaininfo`, did you want to open a ticket?